src/components/Tabs/Tabs.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import PropTypes from 'prop-types';
+import Tab from "./Tab/Tab.js";
+import "./style.scss";
+
+export default function Tabs({tabs, activeTabIndex, onClickTabItem}) {
+  return (
+    <div className="tabs">
+        {tabs.map((tab, i) => (
+          <Tab
+            name={tab.name}
+            tabIndex={i}
+            isActive={activeTabIndex ? i === activeTabIndex: i === 0}
+            onClick={onClickTabItem}
+            key={i}
```

Comment:
Don't forget to avoid using index as a key, you could 'tab.name' as key here because names are unique

Reply:
Done